### ISSUE TITLE
Paying exact amount is impossible

### ISSUE THREAD

**Issue by nwzll** [BoltzExchange/boltz-frontend#193]
Exchanging the exact amount is impossible because boltz asks to round "You send" amount to the nearest 0.0001s

So, paying an exact lightning request is impossible, you can either pay a bit more or a bit less than the amount in payment request.

**Comment by michael1011** [comment]
Yeah this is certainly true and something we have been looking into in the last few weeks.

Btw: to mitigate that for now, you can paste an invoice with any amount in the field and Boltz will accept it and ask for the equivalent onchain coins

@nwzll
